bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -88,20 +90,37 @@ def load_data(input_file, vetoes, ifos, opts, injections=False):
                      'bank': 'bank_chisq',
                      'auto': 'cont_chisq'}
 
-        data[veto_type] = \
-            trigs_or_injs['%s/%s' % (opts.ifo, veto_tags[veto_type])][:]
+        # FIXME: network data contains my_network_chisq
+        chisq_key = 'network/my_network_chisq' if injections \
+            else opts.ifo + '/' + veto_tags[veto_type]
+        # Will not work until single IFO data is in found_missed_file
+        data[veto_type] = trigs_or_injs[chisq_key][:]
 
         # Floor single IFO chi-square at 0.005
         numpy.putmask(data[veto_type], data[veto_type] == 0, 0.005)
 
-        data['dof'] = numpy.unique(
-            trigs_or_injs['%s/%s_dof' % (opts.ifo, veto_tags[veto_type])][:])
+        if not injections:
+            data['dof'] = numpy.unique(
+                trigs_or_injs['%s/%s_dof' % (opts.ifo, veto_tags[veto_type])][:])
```

Comment:
Hard to read this. I suggest defining the key in a separate variable and then using that.

Reply:
Done.